cypress/utils/utils.ts

```diff
@@ -835,6 +838,35 @@ export function createMultipleApplications(numberofapplications: number): Array<
     return applicationList;
 }
 
+export function createMultipleApplicationsWithBSandTags(
+    numberofapplications: number,
+    businessservice?: Array<BusinessServices>,
+    tagList?: Array<Tag>
+): Array<Application> {
+    var applicationList: Array<Application> = [];
+    var tags: string[];
+    var business: string;
+    clickByText(navMenu, applicationInventory);
+    for (let i = 0; i < numberofapplications; i++) {
+        if (!businessservice)
+            businessservice = createMultipleBusinessServices(numberofapplications);
```

Comment:
This line is not needed . Since you are getting BS from argument (from the method calling this function) .
So no need to create BS again here .

Reply:
Thanks for pointing out, Done !